### ISSUE TITLE
Allow different distance metrics in consistency_score

### ISSUE THREAD

**Issue by hoffmansc** [Trusted-AI/AIF360#360]
- [ ] Add args to pass through to `NearestNeighbors` to configure the distance metric
- [ ] Add option to standard scale X before fitting KNN
- [ ] Use `LabelEncoder` for binary y, skip for continuous y, error for other types
    - [ ] Use [type_of_target()](https://scikit-learn.org/stable/modules/generated/sklearn.utils.multiclass.type_of_target.html) to determine y type

**Comment by ifsg331** [comment]
Hi, can I work on this issue?

**Comment by ifsg331** [comment]
Wait sorry can I get assigned to this again?
